src/matrix/sync.rs

```diff
@@ -79,7 +79,8 @@ impl MatrixSyncClient {
 
                     self.current_sync = Some(http_stream.send_request(Request {
                         method: "GET",
-                        path: format!("{}?{}", self.url.path(), self.url.query().unwrap_or("")),
+                        //path: format!("{}?{}", self.url.path(), self.url.query().unwrap_or("")),
+                        path: format!("{}?{}", self.url.path(), ""),
```

Comment:
What happened to the query part?

Reply:
Good catch! This is left over from me trying to figure out why http requests were not correctly being made and I forgot to remove it. I actually noticed this was a problem yesterday when the async / await tests didnt work correctly.